### PR TITLE
Doc update

### LINKED ISSUE #1: Cleaning up the repository
Here is a thread to gather all the different tasks to do now that we have forked the repository before making it public. I want to make sure we didn't miss anything.

- [x] Update the Documentation
- [x] Upload the Wiki and update it
- [x] Remove Eurostag/DTU models
- [x] Fix links in description

### PR DESCRIPTION
**Pull request by MaximeBaudette**
Upload of the rest of the existing documentation

closes #1 

One step closer to #16

### CODE REVIEW

File: docs/community/community-home.rst [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+.. image:: img/smarts-lab-contributors.png
+   :align: center
+
+Other Collaborators
+^^^^^^^^^^^^^^^^^^^
+
+Thanks to the following people who have contributed patches, new code or given us helpful suggestions:
+
+- Dietmar Winkler (HIT, Norway, Dietmar.Winkler@hit.no)
```
Comment (dietmarw) [inline docs/community/community-home.rst]: see above. I clicked to soon

Comment (dietmarw) [inline docs/community/community-home.rst]: And the markup swallowed the rst link syntax. But I guess you can figure that out.

Comment (MaximeBaudette) [inline docs/community/community-home.rst]: Ok I will fix that !

**Comment by MaximeBaudette** [comment]
Do not merge yet people... I forgot to update the readme with corresponding links to the new documentation

**Comment by MaximeBaudette** [comment]
Now it's good, ping @tinrabuzin

**Pull request merged**